numpy/core/getlimits.py

```diff
@@ -372,6 +372,8 @@ class finfo:
     ----------
     bits : int
         The number of bits occupied by the type.
+    dtype : dtype
+        Returns the type.
```

Comment:
I propose: Returns the dtype for which `finfo` returns information. For complex input, the returned dtype is the associated `float*` dtype for its real and complex components.

And then add to Examples:
```python
>>> np.iinfo(np.int32).dtype
dtype('int32')
>>> np.finfo(np.float64).dtype
dtype('float64')
>>> np.finfo(np.complex64).dtype
dtype('float32')
```

Reply:
I'll use the definition you suggest and give examples for finfo(), but I'm not sure if it's a good idea to put an example for iinfo(). For starters, the iinfo() example should be under the examples for iinfo() (I assume that was just a typo on your part), but I also think it's unnecessary. The only confusing part about the dtype attribute is how it works for complex values, so it makes sense to use examples for finfo(), but since iinfo() doesn't deal with complex values, putting an example seems redundant.

Reply:
agreed, no need for an `iinfo` example